tools/SchemaManager.Core/SqlSchemaManager.cs

```diff
@@ -131,13 +128,15 @@ public virtual async Task ApplySchema(string connectionString, Uri server, Mutua
 
             if (availableVersions.First().Id == 1)
```

Comment:
We should not use `First()` or `Last()` here. We have a list, so we know we have indices to perform random access. We can even use the newer syntax `[^1]` to access the last element.